<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zubrilo</class>
 <widget class="QWidget" name="Zubrilo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="katalog">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>679</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>261</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Фильтры</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Язык</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lang_filter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата создания</string>
    </property>
   </widget>
   <widget class="QComboBox" name="date_filter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="use_filter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="info">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>8</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Запустить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>580</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
